core/extensions.js

```diff
@@ -55,7 +55,7 @@ Blockly.Extensions.MUTATOR_PROPERTIES_ =
  * handlers and mutators. These are applied using Block.applyExtension(), or
  * the JSON "extensions" array attribute.
  * @param {string} name The name of this extension.
- * @param {function} initFn The function to initialize an extended block.
```

Comment:
This is invalid syntax according to the compiler.  Should be either {function()} (with any arguments specified) or {Function}.